storage/src/tests/visiting/visitormanagertest.cpp

```diff
@@ -47,7 +47,7 @@ struct VisitorManagerTest : Test {
     std::unique_ptr<TestVisitorMessageSessionFactory> _messageSessionFactory;
     std::unique_ptr<TestServiceLayerApp> _node;
     std::unique_ptr<DummyStorageLink> _top;
-    VisitorManager* _manager;
+    VisitorManager* _manager{};
```

Comment:
This is redundant with the explicit initialization of `_manager` to `nullptr` in the constructor

Reply:
Thanks, fixed